python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -242,6 +242,35 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 2.4 Configure data with OutputFileDatasetConfig objects (Optional)\n",
+    "Intermediate data (or output of a step) is represented by an OutputFileDatasetConfig object. output_data is produced as the output of a step. Optionally, this data can be registered as a dataset by calling register_on_complete. If you create an OutputFileDatasetConfig in one step and use it as an input to another step, that data dependency between steps creates an implicit execution order in the pipeline.\n",
```

Comment:
It may be a good idea to provide a little bit of motivation for this (optional) step. Right now it is not clear when should this step be used.

Reply:
Provided description along with a use case.